src/controllers/events.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable consistent-return */
+const eventModel = require('../models/event');
+
+const getOneEvent = async (req, res) => {
+    const { id } = req.params;
+    try {
+        const event = await eventModel
+            .findById(id)
+            .populate('publisherId')
```

Comment:
Populating the publisher like this will result in a big user object that contains even the hashedPassword property of the user model. to avoid this, use the populate function like this: 
```
const requiredUserField = [
    'id',
    'firstName',
    'lastName',
    'email',
    'profileImage',
];

event = await EventModel.findById(id)
    .populate("publisherId", requiredUserField.join(" "))
    .populate("confirmedVolunteers", requiredUserField.join(" "))
    .populate("invitedVolunteers", requiredUserField.join(" "));


```